server/graphql/v2/enum/PaymentMethodType.ts

```diff
@@ -1,10 +1,10 @@
 import { GraphQLEnumType } from 'graphql';
 
-import { PAYMENT_METHOD_TYPES } from '../../../constants/paymentMethods';
+import { PAYMENT_METHOD_TYPE } from '../../../constants/paymentMethods';
 
 export const PaymentMethodType = new GraphQLEnumType({
   name: 'PaymentMethodType',
-  values: PAYMENT_METHOD_TYPES.reduce((values, key) => {
-    return { ...values, [key]: {} };
+  values: Object.keys(PAYMENT_METHOD_TYPE).reduce((values, key) => {
+    return { ...values, [key]: { value: PAYMENT_METHOD_TYPE[key] } };
```

Comment:
Too keep it retro-compatible, I would suggest:

```suggestion  values: {
      ...PAYMENT_METHOD_TYPES.reduce(
      (values, key) => ({ ...values, [key]: { deprecationReason: 'Please use uppercase values' } }),
      {},
    ),
    ...Object.keys(PAYMENT_METHOD_TYPE).reduce(
      (values, key) => ({ ...values, [key]: { value: PAYMENT_METHOD_TYPE[key] } }),
      {},
    ),
```

Reply:
Thank you for the tip, trying this.